deploy/examples/operator-openshift.yaml

```diff
@@ -681,6 +681,11 @@ spec:
             # Disable automatic orchestration when new devices are discovered
             - name: ROOK_DISABLE_DEVICE_HOTPLUG
               value: "false"
+
+            # Skip watching for Object Bucket resources. If true, the operator won't watch for object bucket resources.
+            # - name: SKIP_OBJECT_BUCKET_WATCH
+            #   value: "true"
```

Comment:
Since this is such a corner case where we want to allow disabling the OBCs, I'd suggest we not put the example settings in operator.yaml or operator-openshift.yaml.

Reply:
agreed.